### PR TITLE
Put content into User Community appendix

### PR DESCRIPTION
**Pull request by dangunter**
Largely by moving text around, I got some content in the User Community WG appendix. The main section became much shorter of course.

I also made one tweak in the "best practices" WG text so my Latex build would succeed, commenting out a citation that pointed nowhere.

@ethanrd please take a look when you have a chance. I included your name as the POC (along with mine).

### CODE REVIEW

File: WSSSPE3/report/wg_appendix_user_community.tex [hunk 1]
```diff
@@ -3,7 +3,9 @@ \section{User Community Working Group Discussion}
 \label{sec:appendix_user_community}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-\todo{add POC here}
+POC:
+Dan Gunter <dkgunter@lbl.gov> and 
+Ethan Davis <edavis@lbl.gov>
```
Comment (ethanrd) [inline WSSSPE3/report/wg_appendix_user_community.tex]: Should be "@ucar.edu"

File: WSSSPE3/report/wg_appendix_user_community.tex [hunk 2]
```diff
@@ -41,13 +93,50 @@ \subsection{Key Next Steps}
...
+\begin{itemize}
+\item Complete and write up a survey of project members, and community members, for successful science software projects
+\item Distill the survey results and document best practices around community engagement
+\end{itemize}
+
 
 \subsection{Description of Funding Needed}
+
+With a small amount of seed funding, it is possible that members of this group or other parties could
+spend the time necessary to devise a survey of existing projects and deploy this, probably including traveli to
```
Comment (ethanrd) [inline WSSSPE3/report/wg_appendix_user_community.tex]: Change "traveli" to "travel"

**Comment by ethanrd** [comment]
@dangunter the split between the main page and the appendix looks really good.

I suggested two very minor edits in line comments. Not sure if that was the best method for that kind of thing on a pull request or not.

**Comment by danielskatz** [comment]
thanks - this looks very nice

**Pull request merged**